build/docker-bake.platforms.hcl

```
@@ -4,6 +4,6 @@
 target "docker-platforms" {
     platforms = [
         "linux/amd64",
-        "linux/arm64"
+        # "linux/arm64"
```

Perhaps we should at least create a ticket for this or document it elsewhere.

```suggestion
        #unsupported due to libcmt "linux/arm64"
```

---

Changed to:
```
# TODO: libarchive13 (required by xgenext2fs in the emulator-devel
# stage) is not available for arm64. We are temporarily disabling this
# platform now, but will come back to it before merging next/2.0 into
# main.
# "linux/arm64"
```

---

Danilo [confirmed](https://github.com/cartesi/rollups-node/issues/439#issuecomment-2150398192) there's an arm64 release for xgenext2fs, so we should just adapt the PR accordingly and close #439.